SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/GridBasedLBStrategy.cpp

```diff
@@ -103,3 +105,13 @@ bool UGridBasedLBStrategy::IsInside(const FBox2D& Box, const FVector2D& Location
 	return Location.X >= Box.Min.X && Location.Y >= Box.Min.Y
 		&& Location.X < Box.Max.X && Location.Y < Box.Max.Y;
 }
+
+const TArray<TPair<const VirtualWorkerId*, const FBox2D*>> UGridBasedLBStrategy::GetVirtualWorkerToCell() const
```

Comment:
Perhaps we can pull "TArray<TPair<const VirtualWorkerId*, const FBox2D*>>" out into a "type"

using LBStrategyRegions = TArray<TPair<const VirtualWorkerId*, const FBox2D*>>

or something similar.